frontend/fingertips-frontend/lib/chartHelpers/preparePopulationData.ts

```diff
@@ -156,9 +160,9 @@ export const createPyramidPopulationDataFrom = (
     year = getLatestYear(areas[0].healthData);
   }
 
-  const pyramidAreas = areas.map((area) =>
-    convertHealthDataForAreaForPyramidData(area, year)
-  );
+  const pyramidAreas = areas.map((area: HealthDataForArea) => {
+    return convertHealthDataForAreaForPyramidData(area, year);
```

Comment:
Apart from typing the area variable, there is no functional change here just syntax - both the old and new implementation behave the same way.